templates/macros/https_params.conf

```diff
@@ -12,9 +12,6 @@ ssl_prefer_server_ciphers off;
 ssl_dhparam {{ nginx.prefix.config }}/dhparam.pem;
 ssl_stapling on;
 ssl_stapling_verify on;
-{% if hsts %}
```

Comment:
Wo kommt die Variable her? Muss die noch irgendwo entfernt werden / aus der Doku raus?

Reply:
Gehört Zeile 5 in dieser Datei dazu und muss die angepasst werden?